Documentation/joss/paper.bib

```diff
@@ -53,6 +53,18 @@ @article{Amann:2016:IFCPL
 
 
 
+@phdthesis {Amann:2018:Diss,
+	author = "Amann, Julian",
+	title = "Eine objektorientierte Sprache zur Einbettung von Interpretationssemantik in digitale Bauwerksmodelle",
```

Comment:
Can you provide the English title in square brackets as well?

Reply:
Done